backend/audit/test_commands.py

```diff
@@ -48,23 +47,31 @@ def test_load_fixtures(self, mock_scan_file):
         # restore the logging override
         logging.disable(logging.ERROR)
 
-    @patch("audit.validators._scan_file")
-    def test_load_fixtures_federal_awards(self, mock_scan_file):
-        """load_fixtures command makes a SAC with federal awards."""
-        # make sure we have at least one user
-        User = get_user_model()
-        user, _ = User.objects.get_or_create(username="test_at_least_one")
-
-        # mock the call to the external AV service
-        mock_scan_file.return_value = MockHttpResponse(200, "clean!")
-
-        call_command("load_fixtures")
-
-        # should be a federal awards ExcelFile for this user
-        files = ExcelFile.objects.filter(
-            user=user, form_section=FORM_SECTIONS.FEDERAL_AWARDS_EXPENDED
-        )
-        self.assertTrue(files)
-
-        # and the associated SAC has `federal_awards` data
-        self.assertTrue(files.first().sac.federal_awards)
+    # This is loading a workbook that does not validate.
+    # However, we now run tests that load and validate (or intentionally
+    # fail to validate) many workbooks.
+    #
+    # Those tests supercede this test.
+    # Therefore, I'm taking this test out, as it is more fragile
+    # than the `pass` and `fail` many-workbook-tests.
+
+    # @patch("audit.validators._scan_file")
```

Comment:
My vote would be to delete this and document it in the PR desc (as opposed to commenting it out) but not a big deal.